### ISSUE TITLE
[Bug][a11y]: ActionGroup should have role="toolbar" as default / selected button should use aria-pressed

### ISSUE THREAD

**Issue by majornista** [adobe/spectrum-web-components#3221]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

ActionGroup

### Expected behavior

Since ActionGroup uses the RovingTabindexController to manage focus using arrow keys to navigate between items, we should add `role="toolbar"` when no other role or selection mode is specified, so that screen readers will switch into forms mode and keyboard navigation will behave as expected. 

Also, when `selects` is `undefined`, selected buttons should use `aria-pressed` rather than `aria-checked`.

### Actual behavior

ActionGroup does not use `role="toolbar"` when no other role or selection mode is specified, so arrow keys will continue to navigate using a screen reader's browse/virtual cursor mode rather than move focus as they should for a toolbar.

Also, selected items with `role="button"` use `aria-checked`, when they should use `aria-pressed`:

See: https://github.com/adobe/spectrum-web-components/blob/123f54c8c3c6e272844c3edcdcc665feb0d38f39/packages/action-group/src/ActionGroup.ts#L271-L300.

### How can we reproduce this issue?

3. Scroll to '....'
4. Check console
5. See error

### Sample code that illustrates the problem

See: https://github.com/adobe/spectrum-web-components/blob/123f54c8c3c6e272844c3edcdcc665feb0d38f39/packages/action-group/src/ActionGroup.ts#L271-L300

**Comment by najikahalsema** [comment]
completed via #3227
